README.md

```diff
@@ -93,6 +99,78 @@ _Requirements_:
 - Start the NextJS web server: ``yarn start``
 - *Note: If you want to update the code in real time, launch with ``yarn dev``*
 
+## 🔧 Configuration
+
+### 🧩  Integrations
+
+Homarr natively integrates with your services. Here is a list of all supported services.
+
+**Emby**
+*Emby does not need an API key.*
+
+**Lidarr**
+*Emby does not need an API key.*
+
+**Plex**
+*Emby does not need an API key.*
```

Comment:
```md
**Emby**
*Emby does not need an API key.*

**Lidarr**
*Emby does not need an API key.*

**Plex**
*Emby does not need an API key.*
```

They all need API keys, they are just not integrated yet.

Reply:
Why are they even added into Homarr if they're not integrated yet...?